threatmodel.md

```diff
@@ -10,3 +10,10 @@ It is assumed that an attacker may perform one or more the following actions:
 
 While it is not always possible to protect against all scenarios, the system should to the extent possible mitigate and/or reduce the damage caused by a successful attack, detect the occurrence of an attack and notify appropriate parties, yet remain usable for parties operating the system.  Furthermore, the system should recover from successful attacks in a way that presents low operational overhead and risk to users.
 
+Attacker Goals:
+1. Trying to have a party install a malicious image under the attackers control.
+2. Trying to have a party install an outdated image.  For example, one with known security vulnerabilities.
+3. Making images unavailable for installation.
```

Comment:
```suggestion
3. Disrupt the verification of image signatures, for example by making the current version of metadata unavailable.
```

Reply:
~~image~~ - artifact

Reply:
This is a bit more specific.  I don't know if it matters, but it reads like your text is a special case of mine...